lib/nexpose.rb

```diff
@@ -57,25 +57,27 @@
 require 'ipaddr'
 require 'json'
 require 'cgi'
+require 'nexpose/api'
+require 'nexpose/json_serializer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.